### PR TITLE
feat: add `github-branch` option to customize the pushed branch

### LINKED ISSUE #163: Allow custom branch to be pushed
Hi guys, thank you for this awesome action!

I'm creating a 'generic' pipeline to be shared across several repositories and I'm receiving the branch name as a parameter and always getting the workflow from the default branch.

But, my problem is that the action gets the branch name from GITHUB_REF, in this part
https://github.com/TriPSs/conventional-changelog-action/blob/9b8c94a880e647cdc3e5071e15e20f0fe210b5ce/src/helpers/git.js#L5-L7
https://github.com/TriPSs/conventional-changelog-action/blob/9b8c94a880e647cdc3e5071e15e20f0fe210b5ce/src/helpers/git.js#L189
Is possible to add a parameter to receive the branch name? I can create a PR with that 

Thank you!

### PR DESCRIPTION
**Pull request by dhpagani**
I have added the `git-branch` as a input, and pass it via parameters to the functions 

solve #163

### CODE REVIEW

File: src/helpers/git.js [hunk 1]
```diff
@@ -2,9 +2,7 @@ const core = require('@actions/core')
 const exec = require('@actions/exec')
 const assert = require('assert')
 
-const { GITHUB_REPOSITORY, GITHUB_REF, ENV } = process.env
-
-const branch = GITHUB_REF.replace('refs/heads/', '')
```
Comment (TriPSs) [inline src/helpers/git.js:7]: Could you re-add this logic when getting the input.

Comment (dhpagani) [reply src/helpers/git.js:7]: I didn't get the reason to do that =/. In the action.yml, it has the definition and the default value for git-branch parameter

**Comment by dhpagani** [comment]
Hi ! Have you seen my message in the review?

**Comment by TriPSs** [comment]
@dhpagani No? I only see my own comment.

**Comment by dhpagani** [comment]
My bad. I forgot to submit the review =/

File: src/helpers/git.js (hunk 1 above)
Comment (dhpagani) [reply src/helpers/git.js:7]: After all that time, i got what you mean about this logic

**Pull request merged**